manifests/2.3.0/opensearch-2.3.0.yml

```diff
@@ -99,3 +99,9 @@ components:
     checks:
       - gradle:properties:version
       - gradle:dependencies:opensearch.version
+  - name: asynchronous-search
+    repository: https://github.com/opensearch-project/asynchronous-search.git
+    ref: '2.3'
```

Comment:
Hey @bharath-techie I dont see 2.3 branch created for `asynchronous-search`, can you please check?

Reply:
Hi, now I've created, can you please retrigger builds again

Reply:
Hey @bharath-techie thanks, tests now pass, but there arise some conflict.

Reply:
Please add `platform` as well. Also duplicate PR #2585 ?